scripts/utils/kakarot.py

```diff
@@ -0,0 +1,282 @@
+import json
+import logging
+import os
+from pathlib import Path
+from typing import Optional, Union
+
+from eth_account import Account as EvmAccount
+from eth_utils.address import to_checksum_address
+from starknet_py.abi import AbiParser
+from starknet_py.contract import Contract
+from starknet_py.net.account.account import Account
+from starknet_py.net.client_errors import ContractNotFoundError
+from starknet_py.net.client_models import Call
+from starknet_py.net.signer.stark_curve_signer import KeyPair
+from starknet_py.serialization import serializer_for_function
+from starkware.starknet.public.abi import get_selector_from_name
+from web3 import Web3
+from web3.contract import Contract as Web3Contract
+
+from scripts.constants import (
+    CHAIN_ID,
+    EVM_ADDRESS,
+    EVM_PRIVATE_KEY,
+    GATEWAY_CLIENT,
+    KAKAROT_ADDRESS,
+    KAKAROT_CHAIN_ID,
+)
+from scripts.utils.starknet import fund_address, get_starknet_account, get_tx_url
+from scripts.utils.utils import hex_string_to_bytes_array
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+def get_contract(contract_path: str):
+    path = os.path.splitext(contract_path)[0]
```

Comment:
it's fine using `os.path` but since you've also imported `pathlib` it's better to stick to one single library